src/components/NurseryWithdrawals/PlantingProgressList.tsx

```diff
@@ -116,18 +116,19 @@ export default function PlantingProgressList({
 
   const onModalSubmit = () => {
     setShowWarningModal(false);
-    setPlantingCompleted(false, true);
+    setPlantingCompleted(false);
   };
 
-  const setPlantingCompleted = (complete: boolean, skipModal?: boolean) => {
-    if (!complete && !skipModal) {
-      const haveStatistics = true;
-      if (haveStatistics) {
-        setShowWarningModal(true);
-        return;
-      }
+  const validateUndoPlantingComplete = () => {
+    const haveStatistics = true;
```

Comment:
`hasStatistics`?

Reply:
This will validate an array of various subzones to see if at least one has statistics, that's why I used plural, but can change it if it's better

Reply:
thanks for clarifying.. makes sense now